src/app/static/src/js/components/runWorkflow/runWorkflowTabs.vue

```diff
@@ -43,8 +42,8 @@
 
 <script lang="ts">
 import Vue from "vue";
-import runWorkflow from './runWorkflow'
-import runWorkflowProgress from './runWorkflowProgress'
+import runWorkflow from './runWorkflow.vue'
+import runWorkflowProgress from './runWorkflowProgress.vue'
```

Comment:
Genuine question: does `.vue` suffix make a difference? Should we be doing it more generally?

Reply:
IDE highlights error without  .vue but can compile and works

However, why not .vue?